lib/Models/PointParameter.js

```diff
@@ -47,12 +47,37 @@ defineProperties(PointParameter.prototype, {
 });
 
 PointParameter.prototype.formatValueAsString = function(value) {
-    if (!defined(value)) {
+    var val = defined(value) ? value : this.value;
```

Comment:
A bit of a nitpick, but you can simplify this to `var val = defaultValue(value, this.value);`  or even `value = defaultValue(value, this.value);`